src/controllers/queryRunner.ts

```diff
@@ -294,6 +294,11 @@ export default class QueryRunner {
 
         // Send the message to the results pane
         this.eventEmitter.emit('message', message);
+
+        // Set row count on status bar if there are no errors
+        if (!obj.message.isError) {
```

Comment:
Will this be correctly cleared if you have a row count from a previously executed query and then run again but an error occurs?